### PR TITLE
Fix mouse scrolling in tmux

### PR DESCRIPTION
**Pull request by programmerdave**

### CODE REVIEW

File: tmux.conf [hunk 1]
```diff
@@ -51,6 +51,10 @@ set-window-option -g window-status-current-fg black
 
 set -g mouse-utf8 on
 set -g mouse on
+bind -n WheelUpPane if-shell -Ft = "#{mouse_any_flag}" "send-keys -M" "if -Ft= '#{pane_in_mode}' 'send-keys -M' 'select-pane -t=; copy-mode -e; send-keys -M'"
```
Comment (jonstorer) [inline tmux.conf]: what does this do?

Comment (programmerdave) [inline tmux.conf]: https://github.com/tmux/tmux/issues/145

it basically goes into copy mode in the pane that your mouse is hovering over and sends an initial scroll command.

**Comment by kylessnell** [comment]
lgtm, although the whole point of tmux is not to use your mouse :wink:

**Comment by programmerdave** [comment]
@kylessnell Yeah I get it but just having to use the keyboard is annoying sometimes when scrolling and I'd rather have the option to use the mouse when I'm scanning through hundreds of lines of output.

**Pull request closed without merging**